tools/sgen/input/FluentUIStyle.yml

```diff
@@ -148,44 +148,61 @@ Opacity:
   size: [ xSmall: 16pt, small: 24pt, medium: 32pt, large: 40pt, xlarge: 52pt, xxlarge: 72pt ]
+  textCalculatedBackgroundColors: [ "NamedColor(cyanBlue10)", 
+                                    "NamedColor(red10)",
+                                    "NamedColor(magenta20)",
+                                    "NamedColor(green10)",
+                                    "NamedColor(magentaPink10)",
+                                    "NamedColor(cyanBlue20)",
+                                    "NamedColor(orange20)",
+                                    "NamedColor(cyan20)",
+                                    "NamedColor(orangeYellow20)",
+                                    "NamedColor(red20)",
+                                    "NamedColor(blue10)",
+                                    "NamedColor(magenta10)",
+                                    "NamedColor(gray40)",
+                                    "NamedColor(green20)",
+                                    "NamedColor(blueMagenta20)",
+                                    "NamedColor(pinkRed10)",
+                                    "NamedColor(gray30)",
+                                    "NamedColor(blueMagenta30)",
+                                    "NamedColor(gray20)",
+                                    "NamedColor(cyan30)",
+                                    "NamedColor(orange30)" ]
+  textColor: $Colors.Foreground.neutralInverted
   textFont: [ xSmall: "Font(9px, regular)", small: "Font(12px, regular)", medium: "Font(13px, regular)", large: "Font(15px, regular)", xlarge: "Font(20px, regular)", xxlarge: "Font(28px, medium)" ]
 
-NoRingAvatarTokens extends AvatarTokens:
-  ringInnerGap: [ xSmall: $Border.size.none, small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none, xlarge: $Border.size.none, xxlarge: $Border.size.none ]
-  ringThickness: [ xSmall: $Border.size.none, small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none, xlarge: $Border.size.none, xxlarge: $Border.size.none ]
-  ringOuterGap: [ xSmall: $Border.size.none, small: $Border.size.none, medium: $Border.size.none, large: $Border.size.none, xlarge: $Border.size.none, xxlarge: $Border.size.none ]
-  ringDefaultColor: $Colors.Neutral.clear
+AccentAvatarTokens extends AvatarTokens:
+  backgroundDefaultColor: $Colors.Brand.primary
+  foregroundDefaultColor: $Icon.accentColor
 
-UnauthenticatedAvatarTokens extends AvatarTokens:
-  backgroundDefaultColor: $Colors.Background.neutral4
-  foregroundDefaultColor: $Colors.Foreground.neutral3
-  ringDefaultColor: $Colors.Background.neutralDisabled
+GroupAvatarTokens extends AvatarTokens:
+  borderRadius: [ xSmall: $Border.radius.small, small: $Border.radius.medium, medium: $Border.radius.medium, large: $Border.radius.large, xlarge: $Border.radius.large, xxlarge: $Border.radius.xlarge ]
+
+OutlinedAvatarTokens extends AvatarTokens:
```

Comment:
Hey @Victa,

Just FYI regarding the colors for fallback styles: I have used the closest colors in the gray pallet to replicate the current behavior of the Avatarview. Here is the conversion from the current palette to the vNext palette:
	 - Gray50 (#F1F1F1) -> grey94 (#F0F0F0)
	 - Gray600 (#404040) -> grey26 (#424242)
	 - Gray500 (#6E6E6E) -> grey42 (#6B6B6B)
	 - Gray200 (#C8C8C8) -> grey78 (#C7C7C7)

Reply:
Oh yeah, that's right. We don't have the latest colors yet. Thanks!